db/migrate/20170202232146_add_option_total_to_options.rb

```diff
@@ -0,0 +1,5 @@
+class AddOptionTotalToOptions < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
These migrations add colums to the same model in the same commit. It can probably be the same migration